README.md

```diff
@@ -1 +1,70 @@
+
+## Interaction with Zeebe broker using zbctl
+- `zbctl` is used to interact with Zeebe. If you cannot install them locally, `zbctl` binary in `bin` directory can be used as an alternative. Each binary should be used in its corresponding operating system.
+
+| Operating System |   zbctl binary   |
+|:----------------:|:----------------:|
+|       Linux      | bin/zbctl        |
+|       OS X       | bin/zbctl.darwin |
+|      Windows     | bin/zbctl.exe    |
+
+
+- Deploy a process: `zbctl --insecure deploy ./bpmn/<process-name.bpmn>`
+- Create an instance for a process: `zbctl --insecure create instance <process-id> --variables <necessary variables>`. The process ID can be checked by either on the command line after deploying the process, or when opening the process' file in Camunda Modeler. Some variables are pre-defined under `./variables`.
+- Create a job worker: `zbctl --insecure create worker <task-definition-type> --handler <command>`. Task definition's type is set in Modeler.
+- Publish a message: `zbctl --insecure publish message <task-definition-type> --correlationKey=<key>`. The correlation key is set in Modeler as a variable (e.g., "orderId"), and "orderId" must be included when creating the process instance.
+
+## Running job workers without zbctl
+`zeebe-job-worker` is a Gradle project which includes pre-defined job workers. To run:
+- Open `zeebe-job-worker` in IntelliJ IDEA.
+- Run `gradle build`.
+    - For Windows, if you want to use IntelliJ IDEA's Terminal, remember to switch to the Ubuntu Terminal before running the command.
+- There is some pre-defined run configurations for job workers available. Select a suitable one (e.g., CartUpdate) and click the Run button to run a job worker.
+
+Each job worker can print one or more output lines to the command line upon whenever it's job worker is needed.
+
+## Full example on setting up the environment and monitor a process instance
+1. Start the environment: `./start.sh`. Open Zeebe Simple Monitor at `localhost:8083` and monitor the status after each step below.
+2. Deploy a process: `zbctl --insecure deploy ./bpmn/multi-instance-process.bpmn`
+3. Create an instance: ```zbctl --insecure create instance multi-instance-process --variables "`jq '.' variables/multi-instance-process/some-in-stock.json`"```. (If jq is not installed, replace the command between " " by the content in `some-in-stock.json`).
+4. Run the predefined job-worker: Open `CartUpdate.kt` of `zeebe-job-worker` in IntelliJ and run the file. The logs should start showing.
+5. Run a job-worker in command line: `zbctl --insecure create worker initiate-payment --handler cat`.
+6. Publish a message: `zbctl --insecure publish message "payment-received" --correlationKey="2"`. (Why 2? Check `some-in-stock.json`)`.
+
```

Comment:
It would be nice if you could just create
* a simple script to deploy all the processes
* other scripts to create instances and trigger the workflows with some input data

So that it will be quicker & easier to run the test bench.

Reply:
Referring to this: https://louhos.atlassian.net/wiki/spaces/VN/pages/2548924417/Elisa+Project+Proposal+-+TestBench
```
The test bench component has the following requirements:
- Deploys BPMN files to Zeebe
- Trigger workflows with test data
```
-> this should be done by the test bench and included scripts instead of being done manually via zbctl I think

Reply:
and at least currently it is not clear in the instructions which variables should be given when creating instances